### ISSUE TITLE
Requesting VisDrone2019 config hyperparameters to compare results as per the paper

### ISSUE THREAD

**Issue by DebanganMandal** [hoiliu-0801/DQ-DETR#21]
As you can see I am trying to reproduce the results from the paper for my own project (Insane paper btw). I have trained the model using the present config/DQ_5scale.py configuration and got mAP results which are lower than that mentioned in the paper. Also the paper doesnt mention any particular information regarding such configurations in their section "Implementation Details". When I say hyperparameters, I am talking about learning rate, weight decay rate, and others, which might affect the result metrics directly. I am attaching image from my terminal noting the results. Please reply with the utmost hurray, it will be very helpfull.

![Image](https://github.com/user-attachments/assets/0941fa11-5e93-4a6f-9a00-f9d5614e683f)

**Comment by zhangchbin** [comment]
Same, cannot reproduce the performance in Visdrone.

**Comment by Jamin-t** [comment]
I have not yet reproduced this code, but based on my initial understanding, if DQ-DETR needs to select the top 1,500 queries, it may first require generating a larger pool of candidate queries, which could lead to higher computational complexity.

I noticed that you have already successfully reproduced the implementation. Would you be willing to share any empirical performance metrics (e.g., inference speed, memory usage, etc.) from your experiments? If you have any insights or data on this, I would greatly appreciate your guidance.

Thank you very much for your time and assistance!

**Comment by 956zjn** [comment]
May I ask which round of assessment result this is
